Lib/Session/Storage/Handler/CassandraSessionHandler.php

```diff
@@ -59,7 +59,7 @@ public function __construct(ClusterInterface $cluster, array $options, LoggerInt
         if (!isset($options['keyspace']) || !isset($options['column_family']) || !isset($options['session_lifetime'])) {
             throw new \InvalidArgumentException('You must provide the "keyspace", "column_family" and "session_lifetime" option for CassandraSessionHandler');
         }
-        $this->options['session_lifetime'] = intval($this->options['session_lifetime'], 10);
+        $this->options['session_lifetime'] = intval($options['session_lifetime'], 10);
```

Comment:
`$this->options` is fully overwritten 2 lines below, so I don't think intval is actually useful (even though it could make sense). Check if that's actually the case and drop it if it's not needed